onnxruntime/core/graph/graph.cc

```diff
@@ -1286,12 +1286,14 @@ Graph::Graph(const Model& owning_model,
       ORT_ENFORCE(p.second, "Duplicate sparse_tensor_initializer: '", tensor->name(), "' Model is invalid.");
     }
 
+#if GOOGLE_PROTOBUF_VERSION < 5026000
```

Comment:
I think this is a good and low risk fix, especially given the fact that sparse initializers are not used (not aware of any usage).

However, I think Clear() itself is not deprecated, so we can leave Clear() out of #ifdef.